GeoHealthCheck/__init__.py

```diff
@@ -27,12 +27,4 @@
 #
 # =================================================================
 
-from util import read
-
-
-def get_package_version(file_):
-    """get version from top-level package init"""
-    return read(file_)
-
-
-__version__ = get_package_version('../VERSION')
+__version__ = '0.8.0dev'
```

Comment:
Should be like:

`__version__ = '0.8.dev0'`

Reply:
Done (commit: 0d2cc5c)